app/package.json

```diff
@@ -12,7 +12,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "datomish-user-agent-service": "0.0.2",
+    "fs-promise": "0.5.0",
     "request": "2.65.0",
+    "thenify": "3.2.0",
```

Comment:
Potentially unrelated, I don't understand when we should add dependencies to app/package.json exactly. Is there a reason why these aren't in the top level package.json?